CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.24.20] - 2022-02-17
+
+### Added
+
+- Fixed issue 126
```

Comment:
This needs to be more verbose here. The possible subheadings are 'added', 'changed' and 'removed'. I'd use the added section to say which How-to guide was added  (see line 29 of index.rst) and 'changed' section to say which hyper link was changed (see line 28 of index.rst). 

The philosophy behind the CHANGELOG is that someone can read it as a standalone and get a summary of what was done in this Pull Request. In this case, saying fixed issue 126 does not help.